dobi.yaml

```diff
@@ -0,0 +1,25 @@
+# ===================================================
+# meta infos
+# ===================================================
+
+meta:
+  project: bb-buildingblock
+
+# ===================================================
+# alias
+# ===================================================
+
+alias=bb-buildingblock-build:
```

Comment:
can those 'bb-buildingblock' be created form that project variable?

Reply:
no, there is no possibility to set config vars in aliases.